### PR TITLE
Fix HPA default policy

### PR DESCRIPTION
**Pull request by liggitt**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1289503
May be related to #6293

### CODE REVIEW

**Comment by liggitt** [comment]
@deads2k PTAL

**Comment by liggitt** [comment]
[test]

File: pkg/cmd/server/bootstrappolicy/policy_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	"github.com/openshift/origin/pkg/api/v1"
+)
+
+func TestOpenshiftRoles(t *testing.T) {
+	roles := GetBootstrapOpenshiftRoles("openshift")
+	list := &api.List{}
+	for i := range roles {
+		list.Items = append(list.Items, &roles[i])
+	}
+	testObjects(t, list, "bootstrap_openshift_roles.yaml")
```
Comment (deads2k) [inline pkg/cmd/server/bootstrappolicy/policy_test.go]: Explain how you created these files in the error message so people know how to do it.

**Comment by deads2k** [comment]
nit. lgtm

**Comment by liggitt** [comment]
[merge]

**Pull request merged**